### DISCUSSION TITLE
Hosted gitlab

### DISCUSSION THREAD

**Discussion by Meier-Link** [rajatkulkarni95/octarine-feedback#7] [Ideas]
Hello,

Thanks for Octarine, which is a truly helpful app for my daily use 😄

As a Gitlab user, My work is stored on the Framasoft gitlab server (reacheable through framagit.org).

In the Git sync service, I selected "Gitlab", then filled the Remote SSH URL with my git@framagit.org:MyLogin/my-project.git url, and unfortunately, it didn't worked with the error message:

> Please recheck your ssh url, it should be in the format of `git@gitlab.com:username/repo.git`.

Can you see to support hostnames different than "gitlab.com"?

Hope you'll add other wonderful stuffs in your product 💪 

Thanks.

**Comment by rajatkulkarni95** [comment]
Hey! So when i first shipped GitSync i kept it simple in terms of validation for showing the button clicks.

However it doesn't really matter, so i can remove the check for gitlab, and just keep it to be starts and ends with "git" for validation since git will anyway do the next step validation.

I'll ship a fix in the next patch this weekend!

Also thank you for the kind words! Keep using the app and sending all sorts of feedback! And spread the word! Tell someone you know to use it as well!

**Reply by rajatkulkarni95** [reply]
Tracking it #8 here.

**Reply by rajatkulkarni95** [reply]
@Meier-Link This is shipped as part of `0.16.4` - [Changelog](https://octarine.app/changelog/0.16.4)
